src/providers/WorkflowCore.Providers.AWS/Services/SQSQueueProvider.cs

```diff
@@ -54,9 +56,9 @@ public async Task<string> DequeueWork(QueueType queue, CancellationToken cancell
 
         public async Task Start()
         {
-            var workflowQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-workflows"));
-            var eventQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-events"));
-            var indexQueue = await _client.CreateQueueAsync(new CreateQueueRequest("workflowcore-index"));
+            var workflowQueue = await _client.CreateQueueAsync(new CreateQueueRequest($"{_queuesPrefix}-workflows"));
```

Comment:
Would you prefer the hyphen be included in the default value as opposed to being hardcoded here?